_posts/en/newsletters/2021-08-11-newsletter.md

```diff
@@ -0,0 +1,168 @@
+  and enhanced support for hardware wallets.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21528][] aims to improve the p2p propagation of full
+  node listening addresses.  Exposure to a diverse set of addresses is important
+  for nodes to be protected against network partitions such as [eclipse attacks][topic eclipse attacks].
+  When Bitcoin Core nodes receive an address message containing 10 or fewer
+  addresses, they forward it to 1 or 2 peers. This is the primary technique
+  used to self-advertise addresses, so sending to peers that would not relay
+  these addresses would effectively stop or "black hole" the propagation through
+  the network. Although propagation failures cannot be prevented in the malicious case, this
+  patch improves address propagation for the honest cases, such as for block-relay-only
+  connections or light clients.
+
+  This update identifies whether or not an inbound connection is a
+  candidate for forwarding addresses based on whether it has sent an address
+  related message over the connection, such as `addr`, `addrv2`, or `getaddr`. This behavior
+  change could be problematic if there is software on the network that
+  relies on receiving address messages but never initiates an address-related
+  message. Therefore, the author took care to circulate this proposed change
+  before it was merged, including posting it to the
+  [mailing list][addrRelay improvements] and researching
+  [other open source clients][addr client research] to confirm compatibility.
+
+- [LND #5484][] allows storing all data in a single external Etcd
+  database. This improves high-availability deployments by making cluster
+  leadership changes instantaneous. The corresponding LND clustering
+  documentation was previously covered in [Newsletter #157][news157 lnd ha].
+
+- [Rust-Lightning #1004][] adds a new event for `PaymentForwarded` that
+  allows tracking when a payment has been successfully forwarded.  Since
+  successful forwarding may earn fees for the node, this allows tracking
+  that income for the user's accounting records.
+
+- [BTCPay Server #2730][] makes the amount optional when generating
+  invoices. This simplifies the payment flow in cases where the operator
+  delegates the choice of the amount to the user, e.g. when topping up
+  an account.
+
+{% include references.md %}
+{% include linkers/issues.md issues="21528,5484,1004,2730,22383" %}
+[C-Lightning 0.10.1]: https://github.com/ElementsProject/lightning/releases/tag/v0.10.1rc2
```

Comment:
The link points to the rc version of 0.10.1

Reply:
Fixed, thanks @shesek!